composeApp/src/desktopMain/kotlin/org/ooni/probe/Main.kt

```diff
@@ -142,6 +144,37 @@ fun main(args: Array<String>) {
                 )
             },
         )
+
+        LaunchedEffect(Unit) {
```

Comment:
Some questions:
- Is it ok to this run this on every app start?
- This will run on the main thread, could it run on the background?
- And could this be inside a function, to avoid having a longer `main`?